Resources/config/cache.yaml

```diff
@@ -0,0 +1,6 @@
+framework:
+  cache:
+    pools:
+      replicas.cache:
```

Comment:
I think it should maybe be `cache.replicas` — all the others (app, system, validator, ...) are that way around.

Reply:
Done